client/README.md

```diff
@@ -38,3 +38,12 @@ The translated parts of the selected code are marked in green.
 
 ## Requirements
 - [Java Runtime Environment (or JDK), 64 bit, version 1.8 or later](https://www.java.com/en/download/)
+
+## Gobra Tools Provider
+The provider of the Gobra Tools can be configured in the Gobra settings.
```

Comment:
Do people know what is meant, when you write "provider of the tools"? Also "Tools" -> "tools", or is it supposed to by a fixed name such as Gobra Server?

Reply:
This applies to both of your comments: I use Gobra Tools as a fixed name (derived from Viper Tools) and includes the dependencies that Gobra IDE (the client) needs. As it is right now, Gobra IDE users are somewhat exposed to that (at least the advanced users) as they can change the URLs in the settings to a different provider. If you do that, you should/have to know what Gobra Tools are.
When opening the settings, users will see the default URL, e.g. "github.com/viperproject/gobra-ide/releases/latest?asset-name=GobraToolsLinux.zip". In order that they understand this URL and/or can adapt it, I've included an explanation of the URL scheme in the README.
I would keep them in this PR but imho this is definitely a valid point that should be discussed as part of the internal testing. I think Viper IDE also allows to configure the provider URL (but doesn't use a special URL scheme) and Prusti only offers a stable/nightly option as they don't think it makes sense for users to create their own IDE dependencies